src/taskgraph/transforms/run/__init__.py

```diff
@@ -27,146 +27,98 @@
 
 logger = logging.getLogger(__name__)
 
-# Fetches may be accepted in other transforms and eventually passed along
-# to a `task` (eg: from_deps). Defining this here allows them to reuse
-# the schema and avoid duplication.
-fetches_schema = {
-    Required("artifact"): str,
-    Optional("dest"): str,
-    Optional("extract"): bool,
-    Optional("verify-hash"): bool,
-}
-
-#: Schema for a run transforms
-run_description_schema = Schema(
-    {
-        Optional(
-            "name",
-            description=dedent(
-                """
-                The name of the task. At least one of 'name' or 'label' must be
-                specified. If 'label' is not provided, it will be generated from
-                the 'name' by prepending the kind.
-                """
-            ),
-        ): str,
-        Optional(
-            "label",
-            description=dedent(
-                """
-                The label of the task. At least one of 'name' or 'label' must be
-                specified. If 'label' is not provided, it will be generated from
-                the 'name' by prepending the kind.
-                """
-            ),
-        ): str,
-        # the following fields are passed directly through to the task description,
-        # possibly modified by the run implementation. See
-        # taskcluster/taskgraph/transforms/task.py for the schema details.
-        Required("description"): task_description_schema["description"],
-        Optional("priority"): task_description_schema["priority"],
```

Comment:
All these keys that previously referenced the `task_description_schema` should continue referencing it moving forward. I think this is where I was saying we'd need to use that `__annotations__` special member variable. Feel free to reach out.

So:
```
Required("description"): task_description_schema["description"]
```

Would become:
```
description: TaskSchema.__annotations__["description"]
```

I think.. some testing / experimentation needed.